src/src_V2/modules/manager/financial/components/payables/Payables.jsx

```diff
@@ -1,302 +1,75 @@
-import { useRef, useState } from 'react';
-import {
-  Box,
-  Table,
-  TableBody,
-  TableCell,
-  TableContainer,
-  TableHead,
-  TablePagination,
-  TableRow,
-  TableSortLabel,
-  Paper,
-  Checkbox,
-  Tooltip,
-} from '@mui/material';
-import { visuallyHidden } from '@mui/utils';
+import { useEffect, useState } from 'react';
 
-import { Container, Layout, NoData } from '../../../../../components';
-import { useHoverTooltip } from '../../../../../hooks';
-import payablesList from '../../mock/payable';
 import { managerSidebarConfig } from '../../../config';
-import styles from './Payables.module.css';
+import payablesList from '../../mock/payable';
+import { Container, Layout } from '../../../../../components';
+import ControlPanel from './control-panel/ControlPanel';
+import PayableTable from './table/PayableTable';
 
 const rowsPerPageOptions = [20, 50, 100];
 
 const columns = [
-  {
-    id: 'paymentId',
-    label: 'Payment ID',
-  },
-  {
-    id: 'tourId',
-    label: 'Tour ID',
-  },
-  {
-    id: 'tourName',
-    label: 'Tour name',
-  },
-  {
-    id: 'supplier',
-    label: 'Supplier',
-  },
-  {
-    id: 'plannedCost',
-    label: 'Planed cost',
-  },
-  {
-    id: 'actualCost',
-    label: 'Actual cost',
-  },
-  {
-    id: 'paid',
-    label: 'Paid',
-  },
-  {
-    id: 'createdDate',
-    label: 'Created date',
-  },
-  {
-    id: 'invoiceId',
-    label: 'Invoice ID',
-  },
-  {
-    id: 'dueDate',
-    label: 'Due date',
-  },
-  {
-    id: 'paymentDate',
-    label: 'Payment date',
-  },
-  {
-    id: 'paymentType',
-    label: 'Payment type',
-  },
-  {
-    id: 'invoiceAttachment',
-    label: 'Invoice Attachment',
-  },
+  { id: 'paymentId', label: 'Payment ID' },
```

Comment:
@aninaslyan jan i think we can create constant.js file for example in finanical folder level and put all constant related to finance there (even if the data is temporary and we could get it from backend in the future) how do you think?